<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="GLOENML-GJUJG3Q" name="com.appModule.Signin_Test" tests="14" failures="1" timestamp="5 Oct 2016 11:35:11 GMT" time="65.345" errors="0">
  <testcase name="verify_DashboardpageRedirection" time="1.314" classname="com.appModule.Signin_Test"/>
  <testcase name="verify_enter_customerID" time="7.328" classname="com.appModule.Signin_Test"/>
  <testcase name="verify_taponLoginbutton" time="7.075" classname="com.appModule.Signin_Test"/>
  <testcase name="verify_tapon_customerID" time="2.310" classname="com.appModule.Signin_Test"/>
  <testcase name="verify_FundTransferpageRedirection" time="0.873" classname="com.appModule.Signin_Test"/>
  <testcase name="verify_UserAccountNumber" time="5.335" classname="com.appModule.Signin_Test">
    <failure type="java.lang.AssertionError" message="expected [50100051926707] but found [50100051926708]">
      <![CDATA[java.lang.AssertionError: expected [50100051926707] but found [50100051926708]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at com.appModule.Signin_Test.verify_UserAccountNumber(Signin_Test.java:100)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
    </failure>
  </testcase> <!-- verify_UserAccountNumber -->
  <testcase name="verify_enter_userPassword" time="7.628" classname="com.appModule.Signin_Test"/>
  <testcase name="verify_tocheckimageconfirmation" time="2.748" classname="com.appModule.Signin_Test"/>
  <testcase name="verify_UserAccountBalance" time="4.336" classname="com.appModule.Signin_Test"/>
  <testcase name="verify_tapon_myAccount" time="7.295" classname="com.appModule.Signin_Test"/>
  <testcase name="verify_tapon_NEFTTransaction" time="3.227" classname="com.appModule.Signin_Test"/>
  <testcase name="verify_tapBeneficiaryDrpDownbox" time="5.375" classname="com.appModule.Signin_Test"/>
  <testcase name="verify_tapon_continueButton" time="8.484" classname="com.appModule.Signin_Test"/>
  <testcase name="verify_BeneficiaryNames" time="2.017" classname="com.appModule.Signin_Test"/>
</testsuite> <!-- com.appModule.Signin_Test -->
